posting/mvcc.go

```diff
@@ -370,7 +370,7 @@ func (c *Cache) set(key []byte, i *CachePL) {
 		return
 	}
 	c.numCacheSave.Add(1)
-	c.data.Set(key, i, 1)
+	c.data.Set(key, i, int64(i.list.ApproximateSize()))
```

Comment:
```suggestion
	c.data.Set(key, i, 0)
```

From the documentation:

```
// To dynamically evaluate the items cost using the Config.Coster function, set
// the cost parameter to 0 and Coster will be ran when needed in order to find
// the items true cost.
```